analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs

```diff
@@ -485,7 +485,7 @@ static IEnumerable<TMessage> ReadProtobuf(string path)
         {
             var builder = SolutionBuilder.Create()
                 .AddProject(AnalyzerLanguage.FromPath(path), outputKind: outputKind)
-                .AddReferences(additionalReferences)
+                .AddReferences(additionalReferences ?? Enumerable.Empty<MetadataReference>())
```

Comment:
I would add this directly in the method to avoid duplication in the calls.

Reply:
It's duplicate and `OldVerifier` is supposed to go away. I'd like to keep `OldVerifier`-specific pain only there to keep the `VerifierBuilder` clean.